src/components.d.ts

```diff
@@ -7007,11 +7007,11 @@ declare namespace LocalJSX {
      */
     interface LimelTextEditorLinkMenu {
         /**
-          * Open state of the dialog
+          * Open state of the link-menu dialog
```

Comment:
Preferably all these documentation changes that have nothing to do with the commit, should be separately commited with a `docs` type. 😊  

This is because: 
- The reviewer will have an easier time figuring out what has been done
- if we ever want to revet a commit in future, we shouldn't lose the changes, just because they were together with another change that altered the functionality.

Reply:
I don't disagree, however those changes need to be in this commit. They are auto-generated and reflect changes made to the code. If we reverted any changes in this commit the documentation would need to be updated again and be part of the same commit.

Reply:
I'm assuming that some changes made in a commit in another PR didn't make their way into the docs in that PR and that's why some of the updates to the docs aren't relevant to this PR.

Reply:
Hm... ah, ok... that's weird that they didn't come here. 🤔 the rebase should update this file properly.


Sorry I commented in this auto-generated file. I just wanted a place to start a thread. 😅 
I was mainly trying to say if there is a docs change, which is not in direct relation to the `fix` or `feat`, then we need to commit it separately.

And also if the docs are auto generated and have nothing to do with your changes, they should also be marked as `docs` 👍

Reply:
There's a PR in that makes a change to the workflow so that we won't have to update this file like this anymore.